AndrewTehan/hometask-2/get_middle_charachter.rb

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+ENTER_STRING = 'Enter string: '
+ERROR_MESSAGE = "String can't be blank"
+EMPTY_STRING = ''
+EXIT = 'exit!'
+
+def run_cli
+  print ENTER_STRING
+  string = gets.chomp
+
+  case string
+  when EXIT then exit
+  when EMPTY_STRING then puts ERROR_MESSAGE
+  else puts get_middle(string)
+  end
+
+  run_cli
+end
+
+def return_by_index(string, index)
+  string[index]
+end
+
+def return_by_range(string, start_index, end_index)
+  string[start_index..end_index]
+end
+
+def get_middle(string)
```

Comment:
Could you look at [ruby style guide](https://github.com/rubocop/ruby-style-guide#accessormutator-method-names)? it's highly recommended to avoid using such prefixes as `get_*`, `set_*`, so let's try to follow it